### DISCUSSION TITLE
Is the plugin available for Android and iOS?

### DISCUSSION THREAD

**Discussion by falapark** [logancyang/obsidian-copilot#165] [Q&A]
Hi,
I have the Samsung Tab S9 and I am not able to use the plugin when using Obsidian. It says that there is no support for it on my device. Is there a workaround for this?

**Comment by logancyang** [comment]
Mobile support is in the roadmap. However it requires some major development both inside and outside this project. So please stay tuned!

**Reply by Stofferdk** [reply]
Any news?
